spec/services/books/search_strategies/s1/strategy_spec.rb

```diff
@@ -115,4 +115,14 @@
       expect((result["hits"]["hits"].select{|hit| hit['_index'].include?('8d50a0af')}).present?).to be_truthy
     end
   end
+
+  context 'fuzzify' do
+    let(:index_names) { [index_name] }
+
+    it 'applies fuzzy modifiers to unquoted words outside of parentheses only' do
+      expect(search_strategy.send(
+        :fuzzify, 'a humongous word "in quotes" (and parentheses) sometimes"(without)"("spaces")'
+      )).to eq 'a humongous~2 word~1 "in quotes" (and parentheses) sometimes~2 "(without)" ("spaces")'
```

Comment:
this seems to be resulting in words within parenthesis not being fuzzy matched, and i'm not sure thats really what we mean to do. i'm not sure what the intent is here though. if we want the parenthesis to be used as the elastic search [precedence grouping](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html#simple-query-string-syntax) then i think we need to find a way to make the words within continue to be fuzzy, maybe add a space around the `(` and `)` symbols would help?

if the issue is that we're unable to search for text excerpts containing literal parens characters, maybe we need to auto escape them or disable the precedence feature?

Reply:
Well, I can give adding spaces after ( and before ) a try...